credentials/apps/api/v2/serializers.py

```diff
@@ -294,13 +294,15 @@ def create(self, validated_data):
 class SignatorySerializer(serializers.ModelSerializer):
     """Serializer for Signatory objects."""
 
+    organization = serializers.CharField(max_length=255, source="organization_name_override", required=False)
```

Comment:
I don't know why do we need the `organization_name_override `, but I hope that u know what to do.

Reply:
Since with LMS this field comes as "organization" and in the model it is called "organization_name_override" we need this.

Reply:
Thank you. Good.